schema.sql

```diff
@@ -0,0 +1,287 @@
+    referral_fee_matching_pool_basis_points INT NOT NULL,
+    referral_fee_public_round_basis_points INT NOT NULL,
+    chef_fee_basis_points INT NOT NULL,
+    total_matching_pool VARCHAR NOT NULL,
+    total_matching_pool_usd DECIMAL(10, 2) NULL,
+    matching_pool_balance VARCHAR NOT NULL,
+    matching_pool_donations_count INT NOT NULL,
+    total_public_donations VARCHAR NOT NULL,
+    total_public_donations_usd DECIMAL(10, 2) NULL,
+    public_donations_count INT NOT NULL,
+    cooldown_end TIMESTAMP NULL,
+    all_paid_out BOOLEAN NOT NULL,
+    protocol_config_provider VARCHAR NULL,
+    FOREIGN KEY (pot_factory_id) REFERENCES pot_factory (id),
+    FOREIGN KEY (deployer_id) REFERENCES account (id),
+    FOREIGN KEY (owner_id) REFERENCES account (id),
+    FOREIGN KEY (chef_id) REFERENCES account (id),
+    FOREIGN KEY (base_currency) REFERENCES account (id)
+  );
+
+-- Table pot_application
+CREATE TABLE
+  pot_application (
+    id SERIAL PRIMARY KEY,
+    pot_id INT NOT NULL,
+    applicant_id VARCHAR NOT NULL,
+    message TEXT,
+    current_status ENUM('Pending', 'Approved', 'Rejected', 'InReview') NOT NULL,
+    submitted_at TIMESTAMP NOT NULL,
+    last_updated_at TIMESTAMP,
+    tx_hash VARCHAR,
+    FOREIGN KEY (pot_id) REFERENCES pot (id),
+    FOREIGN KEY (applicant_id) REFERENCES account (id)
+  );
+
+-- Table pot_application_review
+CREATE TABLE
+  pot_application_review (
+    id SERIAL PRIMARY KEY,
+    application_id INT NOT NULL,
+    reviewer_id VARCHAR NOT NULL,
+    notes TEXT,
+    status ENUM('Pending', 'Approved', 'Rejected', 'InReview') NOT NULL,
+    reviewed_at TIMESTAMP NOT NULL,
+    tx_hash VARCHAR,
+    FOREIGN KEY (application_id) REFERENCES pot_application (id),
+    FOREIGN KEY (reviewer_id) REFERENCES account (id)
+  );
+
+-- Table pot_payout
+CREATE TABLE
+  pot_payout (
+    id SERIAL PRIMARY KEY,
+    recipient_id VARCHAR NOT NULL,
+    amount VARCHAR NOT NULL,
+    amount_usd DECIMAL(10, 2),
+    ft_id VARCHAR NOT NULL,
+    paid_at TIMESTAMP,
+    tx_hash VARCHAR,
+    FOREIGN KEY (recipient_id) REFERENCES account (id)
+  );
+
+-- Table pot_payout_challenge
+CREATE TABLE
+  pot_payout_challenge (
+    id SERIAL PRIMARY KEY,
+    challenger_id VARCHAR NOT NULL,
+    pot_id INT NOT NULL,
+    created_at TIMESTAMP NOT NULL,
+    message TEXT NOT NULL,
+    tx_hash VARCHAR,
+    FOREIGN KEY (challenger_id) REFERENCES account (id),
+    FOREIGN KEY (pot_id) REFERENCES pot (id)
+  );
+
+-- Table pot_payout_challenge_admin_response
+CREATE TABLE
+  pot_payout_challenge_admin_response (
+    id SERIAL PRIMARY KEY,
+    challenge_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    created_at TIMESTAMP NOT NULL,
+    message TEXT,
+    resolved BOOL NOT NULL,
+    tx_hash VARCHAR,
+    FOREIGN KEY (challenge_id) REFERENCES pot_payout_challenge (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+-- Table donation
+CREATE TABLE
+  donation (
+    id SERIAL PRIMARY KEY,
```

Comment:
change to `INT` instead of `SERIAL` so we can set manually and make sure it stays in sync with Donation ID on contract